src/effects/effectmanifest.h

```diff
@@ -82,6 +83,14 @@ class EffectManifest final {
         m_isMixingEQ = value;
     }
 
+    virtual const bool& isWaveformChangingEQ() const {
+        return m_isWaveformChangingEQ;
```

Comment:
> It is probably more universal usable if this is a per parameter option. Did you think about that?

I don't understand. In what situation would that be helpful?

> If we find a broad consensus to ditch this fake feature. It's also a solution.

Huh? This is a really nice feature.